src/daemon/daemon.cpp

```diff
@@ -124,7 +124,7 @@ bool t_daemon::run(bool interactive)
       return false;
     mp_internals->rpc.run();
 
-    daemonize::t_command_server* rpc_commands;
+    daemonize::t_command_server* rpc_commands = NULL;
```

Comment:
True. The sad is, I know all that, but I tend to not remember it when I should, due to being so used to C. I'll make it a unique_ptr once I've double checked it really does leak.